Libraries/LibJS/AST.cpp

```diff
@@ -138,12 +138,28 @@ Value CallExpression::execute(Interpreter& interpreter) const
     arguments.values().append(function.bound_arguments());
 
     for (size_t i = 0; i < m_arguments.size(); ++i) {
-        auto value = m_arguments[i].execute(interpreter);
-        if (interpreter.exception())
-            return {};
-        arguments.append(value);
+        auto value = m_arguments[i].value->execute(interpreter);
         if (interpreter.exception())
             return {};
+        if (m_arguments[i].is_spread) {
+            // FIXME: Support generic iterables
+            Vector<Value> iterables;
+            if (value.is_string()) {
+                for (auto ch : value.as_string().string())
+                    iterables.append(Value(ch));
```

Comment:
Wait, does this make sense? This means that spreading a string will iterate over the individual characters as integers, not as single-character strings.

Reply:
Nope, it doesn't... that's what I get for assuming `Math.max(...'abc')` would return the number, instead of the thing it actually returns -- `NaN`!